client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -8,17 +8,34 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { requestActivityLogs } from 'state/data-getters';
```

Comment:
We should not be using the Activity Log to get a list of backups. There should be a proper, RESTful backups API which gives us a list of backups. If backups have a list of activities related to them, that should be a nested value or endpoint, like `site/1234/backups/5678/activity`

Reply:
This is important so that we and our partners use the same API to list backups. Backups are not a site "activity" even though they may be related to a site activity.